lib/presenter/pages/portfolio/widgets/monetary_infos_crypto_in_tile.dart

```diff
@@ -65,14 +66,8 @@ class MonetaryInfosCryptoInTile extends HookConsumerWidget {
             ),
           ],
         ),
-        IconButton(
-          alignment: Alignment.topCenter,
-          onPressed: () {},
-          icon: Icon(
-            Icons.arrow_forward_ios_sharp,
-            size: 14,
-            color: Colors.grey.shade800,
-          ),
+        ButtonDetailsCrypto(
```

Comment:
É melhor se for possível navegar ao clicar no tile inteiro ao invés de somente no botão de ícone.